src/scripts/load_bash.sh

```diff
@@ -0,0 +1,4 @@
+cd /home/lind/lind_project/tests/applications/bash/
+./bootstrap_nacl || true;
```

Comment:
What's the purpose of the true here? It doesn't seem like there is anything happening with the return value

Reply:
These were to ensure that the script continues to run when there's an error occurs during building (inevitable errors like python has). But it seems that they does not affect our case as you've also said.
Removed.